rpcs3/Emu/system_config.h

```diff
@@ -58,6 +58,7 @@ struct cfg_root : cfg::node
 		cfg::_bool llvm_ppu_jm_handling{ this, "PPU LLVM Java Mode Handling", false }; // Respect current Java Mode for alti-vec ops by PPU LLVM
 		cfg::_int<-1, 14> ppu_128_reservations_loop_max_length{ this, "Accurate PPU 128-byte Reservation Op Max Length", 0, true }; // -1: Always accurate, 0: Never accurate, 1-14: max accurate loop length
 		cfg::_bool llvm_ppu_accurate_vector_nan{ this, "PPU LLVM Accurate Vector NaN values", false };
+		cfg::uint<0, 10000> llvm_ppu_max_precompile{ this, "PPU LLVM Max Precompiled Executables", 10000 };
```

Comment:
If 0 disables it, it should be default value, and max value should be UINT32_MAX because anything less would be too arbitrary to make sense.